### ISSUE TITLE
Filter date 3.1.0 pushed on rubygems is missing a file

### ISSUE THREAD

**Issue by ph** [logstash-plugins/logstash-filter-date#78]
The gems on rubygems seems to be missing a date_jars file? I came accross this issue when testing the docgen branch.

```
rake aborted!
LoadError: no such file to load -- logstash-filter-date_jars
/Users/ph/GOOD_LOGSTASH/vendor/bundle/jruby/1.9/gems/logstash-filter-date-3.1.0/lib/logstash/filters/date.rb:5:in `(root)'
/Users/ph/GOOD_LOGSTASH/tools/logstash-docgen/lib/logstash/docgen/dynamic_parser.rb:1:in `(root)'
```

**Comment by mcandre** [comment]
+1

**Comment by guyboertje** [comment]
The problem is that the gem package task is not dependent on the vendor task

**Comment by suyograo** [comment]
Fixed in 3.1.1
